From: Andrew Straw <strawman@astraw.com>
Date: Fri, 20 May 2011 08:28:55 -0500
Subject: compile pam_cap with -lpam

A similar fix was in Debian but appears to have been accidentally
dropped. Drop this one if or when debian gets it back so we can
directly sync (Closes: #591410).

Last-Update: 2020-02-09
---
 pam_cap/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pam_cap/Makefile b/pam_cap/Makefile
index 56604fd..638d793 100644
--- a/pam_cap/Makefile
+++ b/pam_cap/Makefile
@@ -16,7 +16,7 @@ install: all
 # written (and you know why it fails), email me and explain why. Thanks!
 
 pam_cap.so: pam_cap.o
-	$(LD) -o pam_cap.so $< $(LIBCAPLIB) $(LDFLAGS)
+	$(LD) -o pam_cap.so $< $(LIBCAPLIB) $(LDFLAGS) -lpam
 
 pam_cap.o: pam_cap.c
 	$(CC) $(CFLAGS) $(IPATH) -c $< -o $@
